.github/workflows/test.yml

```diff
@@ -8,12 +8,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - php-versions: 7.3
-            symfony-versions: 3.4.*
-          - php-versions: 7.4
+          - php-versions: 8.0
             symfony-versions: 4.4.*
-          - php-versions: 7.4
```

Comment:
Could we perhaps leave in 8.1 and Symfony 5.4 and 4.4 please?

Reply:
Sorry, i am not sure what you exactly mean. Can you please change the code for this section?

Reply:
Yes of course, I've just pushed the change 👍